applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -391,24 +475,25 @@ int cloud_decode_command(char const *input)
 		return -ENOENT;
 	}
 
-	group_obj = json_object_decode(root_obj, "get");
-	if (group_obj != NULL) {
+	get_obj = json_object_decode(root_obj, "get");
+	set_obj = json_object_decode(root_obj, "set");
+
+	if ((get_obj == NULL) && (set_obj == NULL)) {
+		return -ENOTSUP;
+	}
+
+	if (get_obj != NULL) {
 		cmd_parsed.group = group_get.group;
-		ret = cloud_search_cmd(group_obj, cmd_parsed.group);
-		if (ret) {
-			return ret;
-		}
+		(void)cloud_search_cmd(get_obj, cmd_parsed.group);
```

Comment:
Should the returned error code be checked?

Reply:
it did not seem necessary to me.  it is just searching for a potential command.  current error sources are a null parameter or bad enum value.  i could add an error code check and print an error msg?

it definitely should not return immediately though, as the parsed json obj needs to be deleted.

Reply:
Right, seems reasonable to ignore it